backend/docker-compose.yaml

```diff
@@ -15,3 +15,53 @@ services:
     image: redis:4.0
     ports:
       - "6379:6379"
+  proddev-backend-wsgi:
+    depends_on:
+      - db
+      - redis
+    profiles:
+      - proddev
+    build:
+      context: .
+      dockerfile: Dockerfile
+    ports:
+      - "8001:80"
+    environment:
+      - DATABASE_URL=postgres://postgres:postgres@db:5432/postgres
+      - DJANGO_SETTINGS_MODULE=officehoursqueue.settings.proddev
+    command: sh -c "python manage.py migrate && { DJANGO_SUPERUSER_PASSWORD=root python manage.py createsuperuser --no-input --username root --email root@pennlabs.org; /usr/local/bin/django-run; }"
+  proddev-backend-asgi:
+    depends_on:
+      - db
+      - redis
+    profiles:
+      - proddev
+    build:
+      context: .
+      dockerfile: Dockerfile
+    ports:
+      - "8002:80"
+    environment:
+      - DATABASE_URL=postgres://postgres:postgres@db:5432/postgres
+      - DJANGO_SETTINGS_MODULE=officehoursqueue.settings.proddev
+    command: sh -c "python manage.py migrate && { DJANGO_SUPERUSER_PASSWORD=root python manage.py createsuperuser --no-input --username root --email root@pennlabs.org; /usr/local/bin/asgi-run; }"
+  proddev-frontend:
+    profiles:
+      - proddev
+    build:
+      context: ../frontend
+      dockerfile: ../frontend/Dockerfile
+    ports:
```

Comment:
Nit: frontend local port should not be `8003`, even if its emulating production.

Using 8000 or 8080 is more standard because the frontend would not be an internal service.

Also I have a slight preference for `8080`, `8081`, and `8000` whenever possible and using other ports when these are exhausted.

Reply:
Would
| service | port |
|- | - |
| nginx | 8000 |
| frontend | 8080 |
| backend-wsgi | 8081 |
| backend-asgi | 8082 |

work?

Reply:
yep!